lib/app/shell/shell.dart

```diff
@@ -61,6 +60,7 @@ class _ShellState extends State<Shell> with UiLoggy, WindowListener {
                 AppLocalizations.delegate,
                 GlobalMaterialLocalizations.delegate,
                 GlobalWidgetsLocalizations.delegate,
+                GlobalCupertinoLocalizations.delegate,
```

Comment:
What's this one?

Reply:
The Cupertino widget set mimics Apple UI. Normally we would not use it at all, but I added this just in case we (or any of our dependencies) start depending on any Cupertino widget. It follows the [official example.](https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization)